android/sdl_android/src/main/java/com/smartdevicelink/transport/SdlRouterService.java

```diff
@@ -862,6 +862,14 @@ public void handleMessage(Message msg) {
                     ParcelFileDescriptor parcelFileDescriptor = (ParcelFileDescriptor) msg.obj;
```

Comment:
Not related to this line, but this class. Since this new code is using the alt transport timer I think it's important that it gets updated a bit. I think we should just use the main looper instead of trying to call a different one. So the method should be updated as follows:

```java
    private void startAltTransportTimer() {
        if (altTransportTimerHandler != null && altTransportTimerRunnable != null) {
            altTransportTimerHandler.removeCallbacks(altTransportTimerRunnable);
        }

        altTransportTimerHandler = new Handler(Looper.getMainLooper());
        altTransportTimerRunnable = new Runnable() {
            public void run() {
                altTransportTimerHandler = null;
                altTransportTimerRunnable = null;
                if (!shouldServiceRemainOpen(null)) {
                    closeSelf();
                }
            }
        };
        altTransportTimerHandler.postDelayed(altTransportTimerRunnable, ALT_TRANSPORT_TIMEOUT_RUNNABLE);
    }
```